### ISSUE TITLE
Ongoing Research

### ISSUE THREAD

**Issue by NinaTanneberger** [NinaTanneberger/studio3#17]
Research will be consistently performed during this project in every stage - from marketing to design and straight through to the prototype and final product. Each week I will comment credible sources of research that I applied to my project and comment on them - whether they were useful or not.

**Comment by NinaTanneberger** [comment]
From Lynda Libraries, this [short video](https://www.lynda.com/Business-Project-Management-tutorials/What-agile-project-management/122428/147340-4.html) outlines the basics of Waterfall and mostly Agile project management plans.

I found this useful in determining my project plan as I use a combination of both strategies in my project. 

More about my project plan can be found in the [Project Management Wiki](https://github.com/NinaTanneberger/studio3/wiki/Project-Management).

**Comment by NinaTanneberger** [comment]
Whilst researching Structured Metadata I found this [article ](http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.1.5593&rep=rep1&type=pdf) following a Google Scholar search which was informative on how users interact with metadata. 

The article discusses how bad user interface can result in the user being confused - particularly with how to use metadata in general. This however is based on subjects applying metadata and not simply responding to it. An interesting theory put forward is the ability to flag bad metadata as a user browses the web.

A bit irrelevant for what I want to do with my website but still an interesting look at how metadata interacts with the user which is the area I am most interested in within the web industry.

**Comment by NinaTanneberger** [comment]
As I am due to post one per week of these research findings, I was ill during week 3 and had nothing to post. If i find any relative information for my current task I will post it but it is not a research dense task.

**Comment by NinaTanneberger** [comment]
Following some personal issues during the last week I have not been doing much research, but instead just looking at designs and wireframe drafting. I decided to watch some Ted talks without much expectation when I found this [talk](https://www.ted.com/talks/tim_harford_a_powerful_way_to_unleash_your_natural_creativity/transcript?language=en#t-217283) which struck a chord with me. 

Time Harford discusses a concept coined "slow burn multitasking"; something I have been doing this trimester without realising it - and I believe has been a contributing factor of maintaining my motivation for the project. The concept is that when you work on a project and get stuck, instead of being stuck and procrastinating or giving up, sometimes the solution can be to look at other projects. This trimester whenever I was stuck or needed a break or felt demotivated - I would go and do something creative. I would pick up my camera and take photos, I would edit them carefully, I would play some piano. I found these practices good in maintaining focus on my project rather than just stressing and procrastinating the way I usually would.

A second [video ](https://www.ted.com/talks/cal_newport_why_you_should_quit_social_media?language=en)I looked at was due to my abandonment of Facebook in the last week. Cal Newport talks about the damaging effects this social media platform has on the productive mind - how it promotes addiction and demotes productivity. I always knew Facebook was bad for me but I never realised how much until I went without it. As Cal states in this video - test subjects have shown to be instantly happier following a deletion of their account which is exactly what I experienced. I have been more productive, more energised and more focused.

In relation to the Facebook issue I find Jaron Lanier has a great insight to the social media problem and has written a book on it _en Arguments for Deleting Your Social Media Accounts Right Now_. He discusses how these platforms are designed to addict the users and I found this [interview ](https://www.youtube.com/watch?v=3J1MIDTi9h8&t=335s)with him a great summary of the points he makes on the subject.
